PurchasesTests/Caching/InMemoryCachedObjectTests.swift

```diff
@@ -7,7 +7,7 @@ import Foundation
 import Nimble
 import XCTest
 
-@testable import Purchases
```

Comment:
testable is no longer needed for `Purchases` since it doesn't have swift files, and `@testable` is only for swift